packages/react-reconciler/README.md

```diff
@@ -219,10 +219,16 @@ This is a property (not a function) that should be set to `true` if your rendere
 To implement this method, you'll need some constants available on the _returned_ `Renderer` object:
 
 ```js
+import {
+  DiscreteEventPriority,
+  ContinuousEventPriority,
+  DefaultEventPriority,
+} from './ReactFiberReconciler/src/ReactEventPriorities';
```

Comment:
Hmm. How does this work? It doesn't exist in the open source npm package. Should this suggest to copy-paste from our repo?

Reply:
I'll send a follow-up PR

Reply:
https://github.com/facebook/react/pull/21062